pkg/agent/billing/hostname.go

```diff
@@ -0,0 +1,24 @@
+package billing
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+)
+
+var hostname string
+
+func init() {
+	var err error
+	hostname, err = os.Hostname()
+	if err != nil {
+		hostname = fmt.Sprintf("unknown-%d", rand.Intn(1000))
```

Comment:
I am a bit concerned about this code, what will happen in a very rare but still possible case if we have the same generated hostname for 2 different machines? should it be uuid instead?

Reply:
@sharnoff @Omrigan

Reply:
created an issue out of that.